tests/attacks/poison/test_adversarial_embedding.py

```diff
@@ -0,0 +1,100 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+import numpy as np
+import pytest
+
+from art.attacks.poisoning import PoisoningAttackAdversarialEmbedding
+from art.attacks.poisoning.backdoor_attack import PoisoningAttackBackdoor
+from art.attacks.poisoning.perturbations import add_pattern_bd
+from tests.utils import ARTTestException
+
+logger = logging.getLogger(__name__)
+
+NB_EPOCHS = 3
+
+
+@pytest.mark.skipMlFramework("non_dl_frameworks", "mxnet", "pytorch", "kerastf")
```

Comment:
What is the reason for skipping "kerastf"?

Reply:
I remember initially having issues running `kerastf` on my machine, but it's working now so I'll try it in CI.

Reply:
It seems I have issues with the shape of the output when using kerastf classifier. See the [logs in CI](https://github.com/Trusted-AI/adversarial-robustness-toolbox/runs/1533769285).

Reply:
@ebubae Do you know the reason? Is it correct that it passes for you locally but fails in CI?